modules/serverless-functions-all-values-in-configmap-to-env-variables.adoc

```diff
@@ -5,7 +5,7 @@
 [id="serverless-functions-all-values-in-configmap-to-env-variables_{context}"]
 = Setting environment variables from all values defined in a config map
 
-You can use the following procedure to set an environment variable from all values defined in a config map.
+You can set an environment variable from all values defined in a config map. Values previously stored in a config map can then be immediately used in other parts of the `func.yaml` file.
```

Comment:
I'm not sure if this last sentence is actually true. The `func.yaml` file can be configured to read environment variables from a config map, but that does not mean that these values can be further used in `func.yaml`. Rather, the function can access the values from the config map as environment variables at run time.